code/__HELPERS/see_through_maps.dm

```diff
@@ -0,0 +1,30 @@
+//global statics for the see_through_component coordinate maps
+//For ease of use, include a comment in the shape of the coordinate map, where O is nothing, X is a hidden tile and A is the object
+GLOBAL_LIST_INIT(see_through_maps, list(
+	// X
+	// A
+	"default" = list(
+						list(0, 1, 0)
+	),
+
+	// XXX
+	// XXX
+	// OXO
+	// OAO
+	"pan_shape" = list(
+		list(-1, 3, 0), list(0, 3, 0), list(1, 3, 0),
+		list(-1, 2, 0), list(0, 2, 0), list(1, 2, 0),
+					    list(0, 1, 0)
+	),
+	// X
+	// X
+	// A
+	"default_two_tall" = list(
+					   	list(0, 2, 0),
+					   	list(0, 1, 0)
+	)
+))
+
+#define SEE_THROUGH_MAP_DEFAULT GLOB.see_through_maps["default"]
+#define SEE_THROUGH_MAP_PAN_SHAPE GLOB.see_through_maps["pan_shape"]
+#define SEE_THROUGH_MAP_DEFAULT_TWO_TALL GLOB.see_through_maps["default_two_tall"]
```

Comment:
not sure how I feel about this

Reply:
Afwul? Alternatives are just making the defines straight up the lists, but that's a lot of unnecessary items. Or put them on a singleton datum or something? I don't know a 'non-afwul-solution'

Reply:
why not just use the global?

Reply:
Done!